### PR TITLE
[IMP] project: set default milestone in newly created task in sharing

### PR DESCRIPTION
**Pull request by kcv-odoo**
Before this commit milestone set on task does not set on newly created sub task in project sharing by default and it was setting it by default in backend.

This commit add contect to set milestone by default on newly created subtask from project sharing to be consistant with backend.

task-3716784

### CODE REVIEW

File: addons/project/views/project_sharing_project_task_views.xml [hunk 1]
```diff
@@ -209,7 +209,10 @@
                             <field name="description" type="html" options="{'collaborative': true, 'allowCommandImage': false, 'allowCommandVideo': false, 'allowCommandFile': false}"/>
                         </page>
                         <page name="sub_tasks_page" string="Sub-tasks">
-                            <field name="child_ids" context="{'default_parent_id': id, 'default_partner_id': partner_id, 'form_view_ref' : 'project.project_sharing_project_task_view_form'}">
+                            <field name="child_ids" context="{'default_project_id': project_id, 'default_parent_id': id,
+                                'default_milestone_id': allow_milestones and milestone_id, 'default_partner_id': partner_id,
+                                'form_view_ref' : 'project.project_sharing_project_task_view_form',
+                            }">
```
Comment (pkyriakou) [inline addons/project/views/project_sharing_project_task_views.xml:215]: With the current implementation (by setting the default project_id) the newly created sub-task will show in the kanban view which is not desirable I believe. 
Is it not possible to add this ``'default_display_in_project': False,`` to the context like it's done in the "back-end" version of the project task form view.

**Review by pkyriakou — COMMENTED** [review]
Thank you very much for the update @kcv-odoo :100: 
I left another comment for you to look into :grin:

File: addons/project/views/project_sharing_project_task_views.xml (hunk 1 above)
Comment (kcv-odoo) [reply addons/project/views/project_sharing_project_task_views.xml]: Done! :100:

**Review by pkyriakou — COMMENTED** [review]
Thank you very much for the updates @kcv-odoo :100: LGTM :green_circle: :star:

**Pull request closed without merging**